src/elements/kw-view-feed/kw-view-feed.html

```diff
@@ -171,20 +171,21 @@
                     share="[[selectedShare]]"
                     tombstone$="[[!selectedShare]]"
                     on-post-comment="_onPostComment"
-                    on-remix-action="_onRemixAction"> 
+                    on-remix-action="_onRemixAction"
+                    on-load-comment="_loadMoreData"> 
                     </kw-share-detail>
                 </div>
             </div>
         </div>
-        <kano-modal id="shopDialog" assets-path="../../bower_components/web-components/demo-assets" banner-link="../../bower_components/web-components/demo-assets/avatar/judoka-face.svg">
-            <iron-image sizing="contain" src="http://world.kano.me/assets/projects/overlay-photo.png"></iron-image>
+        <kano-modal id="shopDialog" assets-path="assets" banner-link="assets/avatar/judoka-face.svg">
+            <iron-image sizing="contain" src="assets/projects/overlay-photo.png"></iron-image>
```

Comment:
I think these links need to use the site root `/assets/projects/overlay-photo.png`